crates/ruff/src/settings/flags.rs

```diff
@@ -6,6 +6,12 @@ pub enum Autofix {
     Disabled,
 }
 
+impl Autofix {
+    pub fn is_enabled(self) -> bool {
```

Comment:
Can / should this be `const`?

Reply:
Oh,  it does. by the way, if you want to adapt more convenient ways to do this, I'd like to introduce a macro about it:

https://docs.rs/is-macro/latest/is_macro/derive.Is.html

Reply:
That looks reasonable. We do this a lot especially for flags. Let's maybe start with the various flags and such, and not touch the `ExprKind` and `StmtKind` checks for now.